docs/lib/views.md

```diff
@@ -48,3 +48,22 @@ class DefaultAPIView(AnsibleBaseView):
     all my good stuff
     ...
 ```
+
+
+## Adding extra related fields
+
+There may be cases where the associative router does not provide a way to add specific related views to your view. As a way to help combat this the `AnsibleBaseView` class has a method defined called `extra_related_fields` which returns an empty dict (`{}`) by default. The entries expected in this dictionary are in the format:
+```
+{
+    "string": "url",
+}
+```
+Where string is the name of the field under the related section in the API and URL is the URL to that object. For example, a return like:
+```
+{
+    'authenticators': reverse('user-authenticators-list', kwargs={'pk': obj.pk}),
+}
+```
+Would cause `authenticators` to be listed under the related field and the related URL will be generated from the user-authenticators-list url and should end up as something like `/api/users/:id/authenticators/`.
```

Comment:
> cause `authenticators` to be listed under the related field and the related URL

The language isn't super clear. I would suggest "under the related field" --> "inside of the "related" dictionary in the User model's serializer"